opal/browser/audio/node.rb

```diff
@@ -0,0 +1,187 @@
+module Browser; module Audio; module Node;
+
+  module Pluggable
+    def connect(other_node)
+      `#@native.connect(#{Native.convert(other_node)})`
+    end
+
+    def disconnect(other_node, options = {})
+      other_node = Native.try_convert(other_node)
+      output     = options[:output] || 0
+      input      = options[:input]  || 0
+
+      if options
+        `#@native.disconnect(#{other_node}, #{output}, #{input}) || nil`
+      elsif
```

Comment:
I meant `elsif other_node` and `else`, to flatten the logic, I don't think this is correct syntax.